command/ibft/quorum/params.go

```diff
@@ -0,0 +1,104 @@
+package quorum
+
+import (
+	"errors"
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"os"
+)
+
+const (
+	fromFlag  = "from"
+	chainFlag = "chain"
+)
+
+var (
+	params = &quorumParams{}
+)
+
+type quorumParams struct {
+	genesisConfig *chain.Chain
+	from          uint64
+	genesisPath   string
+}
+
+func (p *quorumParams) initChain() error {
+	cc, err := chain.Import(p.genesisPath)
+	if err != nil {
+		return fmt.Errorf(
+			"failed to load chain config from %s: %w",
+			p.genesisPath,
+			err,
+		)
+	}
+
+	p.genesisConfig = cc
+
+	return nil
+}
+
+func (p *quorumParams) initRawParams() error {
+	if err := p.initChain(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (p *quorumParams) getRequiredFlags() []string {
+	return []string{
+		fromFlag,
+	}
+}
+
+func (p *quorumParams) updateGenesisConfig() error {
+	return appendIBFTQuorum(
+		p.genesisConfig,
+		p.from,
+	)
+}
+
+func (p *quorumParams) overrideGenesisConfig() error {
+	// Remove the current genesis configuration from disk
+	if err := os.Remove(p.genesisPath); err != nil {
+		return err
+	}
+
+	// Save the new genesis configuration
+	if err := helper.WriteGenesisConfigToDisk(
+		p.genesisConfig,
+		p.genesisPath,
+	); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (p *quorumParams) getResult() command.CommandResult {
+	result := &IBFTQuorumResult{
+		Chain: p.genesisPath,
+		From:  common.JSONNumber{Value: p.from},
+	}
+
+	return result
```

Comment:
This can be inlined

Reply:
[Inlined](https://github.com/0xPolygon/polygon-edge/pull/549/commits/86d631d49a1aed37dc888dc6f5b3be768625e6db)